packages/gatsby-theme-newrelic/src/components/Link.js

```diff
@@ -40,18 +35,7 @@ const Link = ({ to, onClick, ...props }) => {
   }
 
   if (isExternal(to)) {
-    const link = isNewRelic(to)
-      ? localizeExternalLink({ link: to, locale })
-      : to;
-    return (
-      <a
-        href={link}
-        onClick={handleExternalLinkClick}
-        target="_blank"
-        rel="noopener noreferrer"
-        {...props}
-      />
-    );
+    return <ExternalLink href={to} onClick={onClick} {...props} />;
```

Comment:
Could we instead do the reverse? I would like to deprecate `ExternalLink` as its got the same functionality as `Link` (as you've already seen). Ideally everything goes through `Link` so that we don't need 2 components. At the very least, could you revert this change and apply the same logic here? I'm ok with duplicate logic in the short term.

Reply:
Could I potentially create a Link folder and then a file with another ExternalLink component or some other name? everything would be served out of `Link`, but it's just so messy with it all being in one file

Reply:
Maybe we should just let `ExternalLink` use `Link`. That might be the best of both worlds until we deprecate everything.

```js
const ExternalLink = ({ href, ...props }) => {
  return <Link to={href} {...props} />
}
```

Thoughts?

Reply:
I'm okay with that! My issue was more towards that `Link` becomes a super dense component - I pushed up the changes to show what I mean! I'm okay with doing it this way, I just wanted to separate out the concerns of external links and non-external links :)

Reply:
Got it! Sorry took me a bit to understand what you meant (too much going on in my brain right now 😬 ). 


I'm ok with this for now. I do agree that maybe we could look at refactoring this a bit, but for now, lets leave it as-is. I don't _think_ we'll need too much more here, but would be good to monitor that over time.